Linux build failing

This is weird, because it worked in the PR (#14). But CircleCI gave
```
/feedstock_root/build_artefacts/snappy_1506339065719/work/snappy-1.1.7/snappy-test.h:106:18: fatal error: zlib.h: No such file or directory
 #include "zlib.h"
```
even though the `zlib` package is present (pulled in as a dependency of something I guess; we should be explicitly including it as a build requirement probably, but that won't change anything here).

I can't debug right now; will look into later.

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -20,6 +20,7 @@ function build() {
         -DCMAKE_INSTALL_PREFIX="$PREFIX" \
         -DCMAKE_PREFIX_PATH="$PREFIX" \
         -DCMAKE_INSTALL_LIBDIR=lib \
+        -DHAVE_LIBZ=FALSE -DHAVE_LIBLZO2=FALSE \
         $extra_args
 
     make -j $CPU_COUNT

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,6 +1,6 @@
 {% set version = "1.1.7" %}
 {% set sha256 = "3dfa02e873ff51a11ee02b9ca391807f0c8ea0529a4924afa645fbf97163f9d4" %}
-{% set build = 0 %}
+{% set build = 1 %}
 
 package:
   name: snappy

